charts/prometheus-blackbox-exporter/templates/role.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.pspEnabled }}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Imho this is a breaking change for users running on older k8s clusters.
Should be implemented via semver version check.

Reply:
Fair enough, will address it through a template in the helper, just like the `prometheus-cloudwatch-exporter` does.